### PR TITLE
Fixes: #2136

### PR DESCRIPTION
**Pull request by sdake**
This is a first take at a table-creating tablegen that categorizes
the output of tablegen.py.  Still needed is the actual text that
goes in the description field.

### CODE REVIEW

File: scripts/tablegen.py [hunk 1]
```diff
@@ -120,17 +131,29 @@ def decode_helm_yaml(s):
...
         code = yaml.load(d_v)
         yaml.explicit_start = True
         yaml.dump(code, sys.stdout, transform=decode_helm_yaml)
 
+        # Order the encoded string dictionary
+        od = collections.OrderedDict(sorted(prdict.items(), key=lambda t: t[0]))
+
+        # Print encoded string dictionary
+        for k, v in od.items():
+            print ("## `%s` configuration options\n" % k)
```
Comment (geeknoid) [inline scripts/tablegen.py]: I'd drop the 'configuration' string in there, it's redundant and causes too much wrapping in the right-hand TOC.

Comment (sdake) [reply scripts/tablegen.py]: done.

**Comment by linsun** [comment]
thanks Steve!

/lgtm

**Pull request merged**